src/main/kotlin/me/zeroeightsix/kami/feature/module/MurderMysteries.kt

```diff
@@ -0,0 +1,124 @@
+package me.zeroeightsix.kami.feature.module
+
+import io.github.fablabsmc.fablabs.api.fiber.v1.annotation.Setting
+import io.github.fablabsmc.fablabs.api.fiber.v1.annotation.Setting.Constrain
+import me.zero.alpine.listener.EventHandler
+import me.zero.alpine.listener.Listener
+import me.zeroeightsix.kami.event.TickEvent
+import me.zeroeightsix.kami.util.Texts
+import net.minecraft.item.Items
+import net.minecraft.text.MutableText
+import net.minecraft.util.Formatting
+
+@Module.Info(
+        name = "MurderMysteries",
+        category = Module.Category.MISC,
+        description = "Get notified when a player holds a dangerous item"
+)
+object MurderMysteries : Module() {
+
+    @Setting(name = "InnocuousItems")
+    private var innocuousItems = true
+    @Setting(name = "WeaponItems")
+    private var weaponItems = true
+    @Setting(name = "ToolItems")
+    private var toolItems = true
+    @Setting(name = "DetectiveItems")
+    private var detectiveItems = false
+
+    @Setting(name = "CooldownOnOutput") // This is how often it will tell the player in chat
+    private var updateEvery: @Constrain.Range(min = 2.0, max = 16.0, step = Double.MIN_VALUE) Int = 16
```

Comment:
It's probably nicer if you also specify a step here. The GUI drag sliders are quite sensitive, so go for a value of around `0.1`